### PR TITLE
feat: implemeted dark mode

### PR DESCRIPTION
**Pull request by Waishnav**
#397 issue solved. 
If you don't like the dark theme colors then do let me know so that I can made further changes.

**Have a look below.**
image

### CODE REVIEW

File: src/components/themeToggle/themeToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+function Toggle() {
+  const [darkToggle, setDarkToggle] = useState(() => {
+    if (localStorage.getItem("theme") === "dark") {
+      return true;
+    }
+    return false;
```
[the remark this answers is not in the document]
Comment (Waishnav) [reply src/components/themeToggle/themeToggle.tsx:8]: I don't think we need to use useEffect ??
Even if you feel it is needed then let me know...

[the remark this answers is not in the document]
Comment (redxzeta) [reply src/components/themeToggle/themeToggle.tsx:8]: Yeah that's fine then

**Review by redxzeta — APPROVED** [review]

**Comment by Waishnav** [comment]
Hey, I was about to change that SVG color in dark mode. cause of some reason I was busy last day so I couldn't able to do it.
Sorry for that.

**Comment by redxzeta** [comment]
> Hey, I was about to change that SVG color in dark mode. cause of some reason I was busy last day so I couldn't able to do it. Sorry for that.

I'll make a new issue for u the.

**Pull request merged**